src/video/resources/playbackRestrictions.ts

```diff
@@ -0,0 +1,45 @@
+import { Base } from '../../base.js';
+import {
+  CreatePlaybackRestrictionParams,
+  ListPlaybackRestrictionsResponse,
+  PlaybackRestrictionResponse,
+  ReferrerDomainRestriction,
+} from '../domain.js';
+
+/**
+ * @private Base playback ID path for the Mux API
+ * */
+const PATH = '/video/v1/playback-restrictions';
+
+/**
+ * @private
+ * Build the base playback ID path for the Mux API
```

Comment:
I think you don't mean playback ID here in the comment, right?

Reply:
woops. correct